src/Parallel/Main.hpp

```diff
@@ -46,8 +46,8 @@ class Main : public CBase_Main<Metavariables> {
 
   /// \cond HIDDEN_SYMBOLS
   /// The constructor used to register the class
-  explicit Main(
-      const Parallel::charmxx::MainChareRegistrationConstructor&) noexcept {}
+  explicit Main(const Parallel::charmxx::
+                    MainChareRegistrationConstructor& /*unused*/) noexcept {}
```

Comment:
This is fixed in develop now :) You canrebase and drop this commit. Jordan fixed in